packages/node_modules/@ciscospark/http-core/src/interceptors/http-status.js

```diff
@@ -39,7 +39,7 @@ export default class HttpStatusInterceptor extends Interceptor {
    * @returns {Promise}
    */
   onResponse(options, response) {
-    if (response.statusCode && response.statusCode < 400) {
+    if (response.statusCode && (response.statusCode < 400 || (response.statusCode === 404 && response.options && response.options.qs && response.options.qs.alternateRedirect))) { // to handle locus redirects
```

Comment:
style nitpick, can you break this into multiple lines to make it easier to parse at a glance?